Libraries/Experimental/SwipeableRow/SwipeableListView.js

```diff
@@ -33,7 +33,7 @@ const {PropTypes} = React;
 type Props = {
   bounceFirstRowOnMount: boolean,
   dataSource: SwipeableListViewDataSource,
-  maxSwipeDistance: number,
+  maxSwipeDistance: number | Function,
```

Comment:
Ok. not a flow user but i have a _getMaxSwipeDistance which has the exact same signature. flow should ensure that always returns a number?